ontology/yaml/resources/HVAC/entity_types/AHU.yaml

```diff
@@ -17,6 +17,23 @@
 ########################
 
 ### Single Zone Units ###
+AHU_HWSC_SFVSC_EFVSC_EDM_SDM_RFC_SFC_FDPM2X_RTM:
+  guid: "ead4d2b6-96f8-44c5-8038-c8114e0da78e"
+  description: "AHU with supply and return fans, fan speed flowrate control heating valve and cooling from DX Unit"
+  is_canonical: true
+  implements:
+  - AHU
+  - HWSC
+  - SFVSC
+  - EFVSC
+  - EDM
+  - SDM
+  - RFC
+  - SFC
+  - FDPM2X
+  - RTM
+  opt_uses:
+  - cooling_percentage_command
```

Comment:
We discourage using single fields on canonical types - what type of cooling element does this unit have? DX or CHW? Why not use CHWSC or DXSC?

Reply:
@cstirdivant - thanks it is a DX unit, I was avoiding using it as I didn't have points for the compressor run and status command. Have changed to DOAS, my mistake. Thank you

Reply:
Thanks for the quick update